### ISSUE TITLE
Is there a way to pick all the icons at once?

### ISSUE THREAD

**Issue by filipeteotonio** [michaelbazos/angular-feather#21]

**Comment by hxutixnnn** [comment]
`import * as icons from 'angular-feather/icons';

@NgModule({
  ...
  imports: [
    ...
    FeatherModule.pick(icons),
    ...
  ],
  ...
})`

**Comment by avmaisak** [comment]
any suggestions?

**Comment by michaelbazos** [comment]
At the moment, there is no way to pick all icons at once.
The rationale behind that choice, is that developers make the effort to create their icon module and pick only icons they are using, so we avoid overloading the resulting JavaScript bundle with unused SVG strings. Currently there are more than 280 icons, and it's very unlikely that even half of them are used in a single application. 
@all What's your thoughts on that matter?

**Comment by daniel-halldorsson** [comment]
I agree with your current approach, seems super unlikely you would need all icons in any app, and by forcing an explicit declaration of which ones are to be included you are avoiding bloating the app with all the icons.

**Comment by avmaisak** [comment]
@daniel-halldorsson 
I think, this is the responsibility of the developer. And library should not limit these possibilities

**Comment by michaelbazos** [comment]
@avmaisak 
Let's say the library developer felt that his responsibility is to offer an API that avoids bad patterns, to the extent possible :)

But, I gave it some more thinking, I can also understand your point of view, so I have reworked the library to also provide a way to import all icons. Here is how to do it:

**icon.module.ts**
```ts
import { NgModule } from '@angular/core';
import { FeatherModule } from 'angular-feather';
import { allIcons } from 'angular-feather/icons';

@NgModule({
  imports: [
    FeatherModule.pick(allIcons)
  ],
  exports: [
    FeatherModule
  ]
})
export class IconsModule { }
```

This is already available on npm, starting from version 6.0.7.

This way to pick all icons will be documented in README too, but with the warning that cherry-picking icons should stay the primary way.

Let me know if that's how you envisioned it.
